Source: John <johnz@posteo.net>
Upsteam: no
Reason: Our go-bindata seems to strip prefixed slash even when -prefix doesn't
end with one.
---
diff --git interfacer/src/browsh/firefox.go interfacer/src/browsh/firefox.go
index 00a9d72..2620522 100644
--- a/interfacer/src/browsh/firefox.go
+++ b/interfacer/src/browsh/firefox.go
@@ -227,7 +227,7 @@ func firefoxMarionette() {
 // Install the Browsh extension that was bundled with `go-bindata` under
 // `webextension.go`.
 func installWebextension() {
-	data, err := Asset("/browsh.xpi")
+	data, err := Asset("browsh.xpi")
 	if err != nil {
 		Shutdown(err)
 	}
